aiohttp/web_reqrep.py

```diff
@@ -70,6 +74,11 @@ def __init__(self, message, payload, transport, reader, writer,
         self._state = {}
         self._cache = {}
         self._task = task
+        if client_max_size:
+            max_size = client_max_size
+        else:
+            max_size = self._CLIENT_MAX_SIZE
```

Comment:
`max_size` could be `None` if user operates with low-level API (without `web.Application`, routers, signals, middlewares and other related stuffs).